lib/swaps/SwapClient.ts

```diff
@@ -10,6 +10,7 @@ enum ClientStatus {
   ConnectionVerified,
   OutOfSync,
   WaitingUnlock,
+  NoChannels,
```

Comment:
I don't think this is the place for this `NoChannels` status, we're never actually setting the status of a client to `NoChannels` and if we did we'd have to alter logic in other places that depends on `ConnectionVerified` to ensure we can make calls to lnd (including calls to open channels).

Reply:
Moved it to errors which makes more sense, thanks Daniel